embassy-rp/src/gpio.rs

```diff
@@ -520,6 +779,35 @@ mod eh02 {
             Ok(self.toggle())
         }
     }
+
+    use core::convert::Infallible;
+
+    impl<'d, T: Pin> embedded_hal_async::digital::Wait for Flex<'d, T> {
```

Comment:
can you add an impl for `Input` as well?

Reply:
I'll add it :+1: